### ISSUE TITLE
Does not work with Xdebug

### ISSUE THREAD

**Issue by efc** [salcode/ssh-into-local-by-flywheel#14]
I love this! Unfortunately, it does not allow Xdebug to work properly. In other words, if I am invoking a WPCLI command from inside the docker container, my breakpoints get hit and I can debug, but if I use this approach to invoke my WPCLI commands from the host Mac, then none of the breakpoints are hit and debugging becomes much more difficult.

Any idea how to resolve that?

**Comment by salcode** [comment]
Hi @efc,

I'm certainly no expert in Xdebug but I suspect the problem is there are two different versions of PHP at play.  The version inside the docker container and the version on your host machine.

It sounds like Xdebug is working for the copy of PHP inside the docker container but not on your host machine.

I can't troubleshoot it at this time but, that's where I'd start.

If you sort this out, I'd love to hear about it and make notes for others who hit this problem.

Thanks.

**Comment by efc** [comment]
Ah, good point, @salcode. I'm sure that is the issue. 

It gets a bit tricky in real life because I debug the non-cli bits with the PHP running in the container, so I need that connected. If I instead connect my code to my host machine's PHP then I'd lose the non-cli debugging. I guess, for now, I'll run the cli in the container when debugging.

**Comment by salcode** [comment]
>  I guess, for now, I'll run the cli in the container when debugging.

While the ability for WP CLI to access things remotely is amazing, I too find that sometimes I need to run commands from within docker.  The other part of this project is meant to help with this.

I don't like having to using the Local by Flywheel GUI to SSH into the docker container.  So I wrote the command `sshlbf` for this project.  Currently, it does require manually creating a `.dockerid` file at the root of the website, but overall I've found it to be a win.
